### PR TITLE
Add detailed info about browser support

### PR DESCRIPTION
**Pull request by battaglr**
References:

- PR #1682.
- [This comment](https://github.com/h5bp/html5-boilerplate/pull/1682#issuecomment-78928913).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,9 +47,22 @@ Choose one of the following options:
...
+## Browser support
+
+* Chrome *(latest 2)*
+* Firefox *(latest 2)*
+* Internet Explorer 8+
+* Opera *(latest 2)*
+* Safari *(latest 2)*
+
+*This doesn't mean that HTML5 Boilerplate cannot be used in older browsers,
+just that we'll ensure compatibility with the ones mention above.*
```
Comment (alrra) [inline README.md]: @battaglr mentioned?

Comment (battaglr) [inline README.md]: @alrra: oops! Fixed.

**Comment by corvannoorloos** [comment]
Nice, thanks!

**Comment by alrra** [comment]
Thanks @battaglr!

Also, thanks @callumlocke for bringing this up!

**Comment by alrra** [comment]
__edit:__ Also, thanks @corvannoorloos for bringing this up!

**Comment by battaglr** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (whardson) [inline README.md]: Hello!

 

New message, please read <http://radek.pomorze.pl/several.php?c>

 

win@affordca.com

**Pull request closed without merging**